src/sidebar/util/thread-annotations.js

```diff
@@ -5,9 +5,23 @@ import { generateFacetedFilter } from './search-filter';
 import filterAnnotations from './view-filter';
 import { shouldShowInTab } from './tabs';
 
+/** @typedef {import('../../types/api').Annotation} Annotation */
 /** @typedef {import('./build-thread').Thread} Thread */
 /** @typedef {import('./build-thread').Options} BuildThreadOptions */
-/** @typedef {import('../store/modules/selection').ThreadState} ThreadState */
+
+/**
```

Comment:
`ThreadState` is now the purview of this utility module instead of the store, which makes sense, I think...